stages/org.osbuild.update-crypto-policies

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python3
+"""
+Sets the policy applicable for the various cryptographic back-ends,
+such as SSL/TLS libraries.
+
+This stage calls `update-crypto-policies` to set the system's
+cryptographic policy.
+
+Notes:
+  - Requires 'chroot' in the buildroot.
+  - Runs the 'update-crypto-policies' script from the image in the chroot.
+"""
+
+
+import subprocess
+import sys
+
+from osbuild import api
+
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "description": "Sets the current policy and overwrites the config file",
+  "required": [ "policy" ],
+  "properties": {
+    "policy": {
+      "type": "string",
+      "minLength": 1,
+      "description": "The policy to be applied."
+    }
+  }
+}
+"""
+
+
+def main(tree, options):
+    policy = options.get("policy", "")
+    if policy == "":
+        return 1
```

Comment:
nitpick: checking if the value was provided / if it is empty should not be needed any more.